parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/lib.rs

```diff
@@ -624,10 +628,14 @@ mod benches {
 		[pallet_timestamp, Timestamp]
 		[pallet_collator_selection, CollatorSelection]
 		[cumulus_pallet_xcmp_queue, XcmpQueue]
-		// XCM
```

Comment:
@svyatonik 
Slava, cool, thank you, somewhere also exists issue for this (I will find),

but please, wait tomorrow with merge , I am almost done with upgrade with merge and master-like build,
and this xcm benchamarks needs to be here,
because in master version it is used and weights are generated for xcm pallet
https://github.com/paritytech/cumulus/blob/master/parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/lib.rs#L471-L475
https://github.com/paritytech/cumulus/blob/master/parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/xcm_config.rs#L190-L194

Reply:
@svyatonik 
Slava, I am done with upgrade, please, rebase/merge this on actual `bridge-hub-rococo-wococo` branch

Reply:
Thanks, I've rebased, but now I'm getting:
```
error[E0046]: not all trait items implemented, missing: `ReachableDest`
   --> /home/svyatonik/.cargo/git/checkouts/polkadot-4038f27d5e4ea2e8/37468ca/runtime/polkadot/src/xcm_config.rs:358:1
    |
358 | impl pallet_xcm::Config for Runtime {
    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ missing `ReachableDest` in implementation
    |
    = help: implement the missing item: `type ReachableDest = Type;`
```
when doing `reset && cargo build --profile production --features runtime-benchmarks -p bridge-hub-rococo-runtime`. Looks like `polkadot-service/runtime-benchmarks` isn't enabled, but I can't find where it comes from.